### DISCUSSION TITLE
How to add mathml code to the document?

### DISCUSSION THREAD

**Discussion by mwashambw** [dolanmiu/docx#2398] [Q&A]
If you take mathml code and you paste in word document it render the formula written in that mathml code.My question is how can I do the same programmatically using docx library?

**Comment by dolanmiu** [comment]
Not possible I am afraid. `docx` does not parse it, that would mean implementing a tokeniser, parser, and transpiler, which is out of scope for this library

read the documentation here:

https://docx.js.org/#/usage/math?id=math
